### PR TITLE
Reformat Contributing

### PR DESCRIPTION
**Pull request by GNosii**
#### Description: 

It's the time of my triyearly PR! This pull request reformats the `CONTRIBUTING.md` file by:
- Fixing grammar errors.
- Simplifying rules, making them easier to understand.
- Simplifying sections, removing redundant content.
- Make use of new GFM features like footnotes.
- Remove commented out section (pre-crowdin translation)
____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [x] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: .github/CONTRIBUTING.MD [hunk 1]
```diff
@@ -1,60 +1,52 @@
...
-
-# Pull & Code Requirements    
-## Pull-Request Requirements
-1. Keep your commit message(s) self-explanitory, description should be a brief explanation and include an issue reference (if applicable.).   
-2. Pull Requests should be simple and straight-forward. One pull request for one feature. Changed Files should only show a difference where code has been removed or added, no re-formatting of entire files.   
-3. Test everything you've touched before pull-requesting. (Exception being language files, we don't expect you to check all lang strings.)    
-
-## General Code Requirements
-> With the inclusion of the Maven build system, Towny also makes use of a [EditorConfig](https://editorconfig.org) file. IDEs that support it should automatically follow parts of this section.
+Thanks for deciding to contribute to Towny! We really appreciate contributions, but to maintain everything cool around here, we need to enforce some guidelines to keep the project as efficient as possible. :shipit:
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:2]:
```suggestion
Thanks for deciding to contribute to Towny! We really appreciate contributions, but to keep everything running smoothly and get contributions accepted in a timely manner, we need to enforce some guidelines. :shipit:
```

File: .github/CONTRIBUTING.MD [hunk 2]
```diff
@@ -1,60 +1,52 @@
...
+You can always ask for help on your contribution in the [Towny Discord](https://discord.gg/gnpVs5m).    
+
+## Before contributing
+- Make sure to read the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md).
+- By contributing, you agree to distribute your changes under the Towny license and all its terms.
+
+If you do not agree with the terms of our license, do not contribute to the Towny project.
+
+# About pull requests
+1. Keep your commit messages self-explainatory, the description should be a brief explanation and include an issue reference if applicable.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:15]:
```suggestion
1. Keep your commit messages self-explanatory. Pull requests will have their Commits squashed upon merge but the commit messages will be summarized.
```

File: .github/CONTRIBUTING.MD [hunk 3]
```diff
@@ -1,60 +1,52 @@
...
+2. Pull requests should be straight-forward and focus only on one feature. Changed files should only show a difference where code has been actually modified.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:16]:
```suggestion
2. Pull Requests forms should be filled out to the best of your abilities: descriptions, changes and issue references where applicable.
```

File: .github/CONTRIBUTING.MD [hunk 4]
```diff
@@ -1,60 +1,52 @@
...
+3. Test your changes before sending them in! You should look for bugs, test configuration values, commands, and anything you've touched!
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:17]:
```suggestion
3. Your pull request should address a single topic, or be as closed in scope as you can reasonably make it. Small pull requests are easier to review and more predictable to test.
4. Test your changes before sending them in! You should look for bugs, test configuration values, commands, and anything you've touched!
```

File: .github/CONTRIBUTING.MD [hunk 5]
```diff
@@ -1,60 +1,52 @@
...
-1. Prevent Class Not Found errors    
+1. You must use tabs for indentation. 
+2. Keep your environment in Java 16-17. The code will automatically be converted into code compatible with Java 8 by [Jabel](https://github.com/bsideup/jabel).[^jabel-warning]
+3. Avoid using wildcard imports.
+4. Do not format entire class files.   
+   - All IDEs are different, we do this to keep consistency in the codebase.
+   - This also keeps code differences in GitHub clean and simple.
+5. Do not use NMS (`net.minecraft.server`)[^nms], or reflection. We aim at only using the Spigot API.
+6. Do not directly interact with the Minecraft protocol.
+   - This includes the usage of ProtocolLib or Netty.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:30]:
```suggestion
5. Do not make unnecessary changes.
    - This includes whitespace changes.
    - We want to have as clean a diff as possible.
6. Do not use NMS (`net.minecraft.server`)[^nms], or reflection. We aim to use only the Spigot/Bukkit API, with some additional systems like PaperLib.
7. Do not directly interact with the Minecraft protocol.
   - This includes the usage of ProtocolLib or Netty.
```

File: .github/CONTRIBUTING.MD [hunk 6]
```diff
@@ -1,60 +1,52 @@
...
+   - Even if you believe it can benefit the project, please ask the maintainer first.
+4. Do not embed dependencies directly into Towny.
+   - All dependencies should use the `provided` scope instead. This means that the dependency will be provided by the plugin JAR (e.g. Vault), or the server JAR (e.g. Adventure).
+5. Do not add closed source dependencies.
+   - All dependencies should have open and auditable source code.
+
+# About translation
+All translation on the core Towny plugin is done via [Crowdin](https://crowdin.com/project/townyadvanced).
+
+Strings are formatted using [Java's `java.util.Formatter`](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/util/Formatter.html) allowing specification of variable order, and more.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:49]:
```suggestion
Strings are formatted using [Java's `java.util.Formatter`](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/util/Formatter.html) allowing specification of variable order, and more. If you find that a line has multiple variables which would be in a different order in your language, you can change them to `%Variable number$s` to change their order. 
An example of this: `%4$s` will **always use the 4th variable**, no matter where it is placed on the line.
```

**Review by LlmDl — CHANGES_REQUESTED** [review]
I've left some suggestions you can add/batch in.

File: .github/CONTRIBUTING.MD [hunk 7]
```diff
@@ -1,60 +1,60 @@
...
+
+Contributions are welcomed via GitHub Pull Requests.
+You must fork this repository and then open it via your IDE.   
+You can always ask for help on your contribution in the [Towny Discord](https://discord.gg/gnpVs5m).    
+
+## Before contributing
+- Make sure to read the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md).
+- By opening an pull request, you:
+    - represent that you have the right to waive your changes' copyright and any related rights.
+    - acknowledge that the TownyAdvanced organization is allowed to use or modify your changes in any way it sees fit and in accordance with the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md)
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:12]: This has to be redone so that there is no change from the original wording.

File: .github/CONTRIBUTING.MD [hunk 8]
```diff
@@ -1,60 +1,60 @@
...
+
+If you do not agree with the terms of our license, **do not contribute to the Towny project**.
+
+# About pull requests
+> Pull requests will have their commits squashed upon merge, commit messages will be summarized.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:17]:
```suggestion
> Pull requests will have their commits squashed upon merge, in doing so your various Commit Messages will be amalgamated into one Commit Message. Consider this while you create your individual commits.
```

File: .github/CONTRIBUTING.MD [hunk 9]
```diff
@@ -1,60 +1,60 @@
...
-## Pull-Request Requirements
-1. Keep your commit message(s) self-explanitory, description should be a brief explanation and include an issue reference (if applicable.).   
-2. Pull Requests should be simple and straight-forward. One pull request for one feature. Changed Files should only show a difference where code has been removed or added, no re-formatting of entire files.   
-3. Test everything you've touched before pull-requesting. (Exception being language files, we don't expect you to check all lang strings.)    
-
-## General Code Requirements
-> With the inclusion of the Maven build system, Towny also makes use of a [EditorConfig](https://editorconfig.org) file. IDEs that support it should automatically follow parts of this section.
+Thanks for deciding to contribute to Towny! We really appreciate contributions, but to keep everything running smoothly and get contributions merged in a timely manner, we need to enforce some guidelines. :shipit:
+
+Contributions are welcomed via GitHub Pull Requests.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:4]:
```suggestion
Contributions are welcome via GitHub Pull Requests.
```

File: .github/CONTRIBUTING.MD [hunk 10]
```diff
@@ -1,60 +1,60 @@
...
+You must fork this repository and then open it via your IDE.   
+You can always ask for help on your contribution in the [Towny Discord](https://discord.gg/gnpVs5m).    
+
+## Before contributing
+- Make sure to read the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md).
+- By opening an pull request, you:
+    - represent that you have the right to waive your changes' copyright and any related rights.
+    - acknowledge that the TownyAdvanced organization is allowed to use or modify your changes in any way it sees fit and in accordance with the [Towny license](https://github.com/TownyAdvanced/Towny/blob/master/LICENSE.md)
+
+If you do not agree with the terms of our license, **do not contribute to the Towny project**.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:14]:
```suggestion
If you do not agree with the terms of our license, do not contribute to the Towny project.
```

File: .github/CONTRIBUTING.MD [hunk 11]
```diff
@@ -1,60 +1,60 @@
...
+
+# About pull requests
+> Pull requests will have their commits squashed upon merge, commit messages will be summarized.
+
+1. Keep your commit messages self-explanatory. Keep them short but informative!
+2. Pull requests should address a single topic, or be as closed in scope as you can reasonably make it. Small pull requests are easier to review and thus, can result in an faster merge.
```
Comment (LlmDl) [inline .github/CONTRIBUTING.MD:20]:
```suggestion
2. Pull requests should address a single topic, or be as closed in scope as you can reasonably make it. Small pull requests are easier to review and thus, can result in a faster merge.
```

File: .github/CONTRIBUTING.MD (hunk 7 above)
Comment (GNosii) [reply .github/CONTRIBUTING.MD:12]: Understood!

**Pull request closed without merging**